package.json

```diff
@@ -22,9 +22,12 @@
     "test": "jest"
   },
   "devDependencies": {
+    "@rollup/pluginutils": "^4.1.2",
     "@types/jest": "^26.0.23",
     "@types/node": "^15.12.1",
+    "@webassemblyjs/wasm-parser": "^1.11.1",
```

Comment:
Why is there a @webassemblyjs/wasm-parser ?

Reply:
To verify that changes do not introduce any regression, you need to run tests on the whole repo. This is done using `npm test`. This repo also contains the `vite-plugin-load-wasm` and it refers to `@webassemblyjs/wasm-parser`:
https://github.com/originjs/vite-plugins/blob/main/packages/vite-plugin-load-wasm/src/parser.ts#L1

If desired, I can remove this reference to `@webassemblyjs/wasm-parser` from my pull-request. However, without this addition, contributors won’t be able to run the repository’s test suite using `npm test`.

BR, Martin

Reply:
This project is monorepo, you can use `yarn test` instead.
Thanks for your PR!